modules/fedimint-wallet-common/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub mod txoproof;
 pub const KIND: ModuleKind = ModuleKind::from_static_str("wallet");
 pub const MODULE_CONSENSUS_VERSION: ModuleConsensusVersion = ModuleConsensusVersion::new(2, 1);
 
-pub const CONFIRMATION_TARGET: u16 = 10;
+pub const CONFIRMATION_TARGET: u16 = 1;
```

Comment:
Can we also update this?

https://github.com/fedimint/fedimint/blob/6dd24f54dbad66edf2c99529523f5e3d2110e175/docs/wallet_module.md?plain=1#L25

Reply:
Oh also, can you add a comment above this constant mentioning why the target is 1? I think just a brief mention of lack of RBF support and federation UTXOs being a shared resource would be useful.

Reply:
Good call 👍